docs/content/usage/management.md

```diff
@@ -13,12 +13,16 @@ always) Git repositories.
 
 Hermit retrieves package manifests from various locations, including Git
 repositories. It will periodically sync these repositories to your system to
-ensure you have the most up to date manifests, but you may also force this by running:
+ensure you have the most up to date manifests. Similarly, hermit periodically 
+updates itself. Hermit and package manifest updates may also be forced by running:
```

Comment:
`hermit update` will actually update all channels now FWIW

Reply:
just to clarify: sync & update are the same, yeah?
Will add a note wrt to channels.

Reply:
ok, updated. hope this ok for you.